### PR TITLE
add jquery-nice-select@1.1.0 w/ git auto-update

### LINKED ISSUE #9988: [Request] Add hernansartorio/jquery-nice-select
**Library name:** hernansartorio/jquery-nice-select
**Git repository url:**
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by hare1039**
close #9988, cc @hernansartorio

@pvnr0082t Please help me review this PR. Thanks.

Pull request for issue: #9988
Related issue(s): #9592 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/hernansartorio/jquery-nice-select
 * Official website (optional, not the repository): http://hernansartorio.com/jquery-nice-select/
 * NPM package url (optional): https://www.npmjs.com/package/jquery-nice-select
 * License and its reference: MIT, 
ref: https://github.com/hernansartorio/jquery-nice-select/issues/37
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 361
   - Count of watchers: 21
   - Count of forks: 67
 * NPM download stats (optional): 
   - Downloads in the last day: 2 
   - Downloads in the last week: 36
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-nice-select/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/hernansartorio/jquery-nice-select.git"
+  },
+  "author": {
+    "name": "Hernán Sartorio",
+    "url": "https://github.com/hernansartorio"
+  },
+  "license": "ISC",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-nice-select/package.json]: @hare1039 You should try to confirm the license with author like https://github.com/hernansartorio/jquery-nice-select/issues/31 but there is no response. Try to open an issue to discuss it. Thanks.

File: ajax/libs/jquery-nice-select/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "files": [
+          "**/jquery.nice-select.*"
+        ]
+      },
+      {
+        "basePath": "css",
+        "files": [
+          "**/*"
+        ]
+      }
```
Comment (pvnr0082t) [inline ajax/libs/jquery-nice-select/package.json]: @hare1039 
1. I don't think we need `style.css` 
2. it's a little bit not safe or no sense to use `**/*` under a non-dist folder. 
3. It's not good to use `js` and `css` as basePath.  

Let's leave basePath blank and combine these two using `+(js|css)/*nice-select*`

**Comment by hare1039** [comment]
@pvnr0082t I opened another issue on https://github.com/hernansartorio/jquery-nice-select/issues/37 wait for reply.
I also change the auto-update syntax to `+(js|css)/*nice-select*`, and `filename` to `js/jquery.nice-select.min.js` to pass the npm test.

Please review this PR again. Thank you.

**Comment by PeterDaveHello** [comment]
You should confirm that if there are duplicated PR or issue exist like #9592 before doing new commit and PR.

**Comment by hare1039** [comment]
@pvnr0082t I minify the css file. 
The author said the library is license under MIT license [ref](https://github.com/hernansartorio/jquery-nice-select/issues/37), so I also changed the `license` to `MIT` in package.json.
Please review this PR again. Thank you.

**Comment by hare1039** [comment]
@pvnr0082t OK, I amend the commit message. Please review this PR again.

**Comment by pvnr0082t** [comment]
LGTM
@cdnjs/intern2 Please help review this PR, thank you.

**Comment by PeterDaveHello** [comment]
1. Don't make hyperlink to issues/PRs by yourself like https://github.com/hernansartorio/jquery-nice-select/issues/37
2. You should send a PR to help correct the license.

**Comment by hare1039** [comment]
Got it. I will be more careful next time.

**Pull request merged**